pkg/local_object_storage/blobstor/put.go

```diff
@@ -89,6 +97,10 @@ func (b *BlobStor) PutRaw(addr *addressSDK.Address, data []byte, compress bool)
 		return new(PutRes), nil
 	}
 
+	if compress {
```

Comment:
is compressing small objects expected to stay that way?

Reply:
Yes, there are no problems here.